LastPush/CanvasGapsGenerator/src/main/kotlin/jetbrains/kotlin/course/last/push/Main.kt

```diff
@@ -69,18 +69,12 @@ fun repeatHorizontally(pattern: String, n: Int, patternWidth: Int): String {
         sb.append(currentRow.repeat(n))
         sb.append(newLineSymbol)
     }
-    return sb.toString()
+    return sb.toString().dropLastWhile { it.toString() == newLineSymbol }
```

Comment:
I'd expect that it will be handled on the tests' side since we don't explain this to the student in the tasks